src/tests/compiler/clvm.rs

```diff
@@ -213,3 +214,19 @@ fn test_random_int_just_the_conversion_functions_and_no_other_things_from_the_st
         }
     }
 }
+
+#[test]
+fn test_sha256_tree_hash() {
+    let filename = "resources/tests/assert.clvm.out";
+    let assert_clvm_compiled = fs::read_to_string(filename).expect("should exist");
+    let parsed = parse_sexp(
+        Srcloc::start(filename),
+        assert_clvm_compiled.as_bytes().iter().copied(),
+    )
+    .expect("should parse");
+    let hash_result = Bytes::new(Some(BytesFromType::Raw(sha256tree(parsed[0].clone())))).hex();
+    assert_eq!(
+        hash_result,
+        "156e86309040ed6bbfee805c9c6ca7eebc140490bd1b97d6d18fb8ebc91fd05a"
```

Comment:
it's not so easy to tell that this is correct by inspecting the test

Reply:
Hmm true.  I do wonder how to communicate things like this.